pages/dkp/kaptain/2.0.0/install/air-gapped-dkp/index.md

```diff
@@ -51,14 +70,14 @@ Use the following instructions to install Spark Operator from Kommander Catalog
 
 ### Load the Docker images into your Docker registry
 
-1. Download the image bundle file:
+1.  Download the image bundle file:
 
-   - Download `kaptain_air_gapped.tar` that will contain the required artifacts to perform an air-gapped installation.
-   - (Optional) Download the custom image artifacts `kaptain_air_gapped_cpu.tar` or `kaptain_air_gapped_gpu.tar` based on whether you need CPU or GPU for your workloads.
+    - Download `kaptain_air_gapped.tar` that will contain the required artifacts to perform an air-gapped installation.
+    - (Optional) Download the custom image artifacts `kaptain_air_gapped_cpu.tar` or `kaptain_air_gapped_gpu.tar` based on whether you need CPU or GPU for your workloads.
 
-2. Place the bundle in a location where you can load and push the images to your private Docker registry.
+1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
 
-3. Ensure you set the `REGISTRY_URL` and `AIRGAPPED_TAR_FILE` variable appropriately, then use the following script to load the air-gapped image bundle:
+1.  Ensure you set the `REGISTRY_URL` and `AIRGAPPED_TAR_FILE` variable appropriately, then use the following script to load the air-gapped image bundle:
```

Comment:
In addition for DKP 2.2 you need to download the application bundle and the chart bundle. Extract the application bundle to the location referenced in the Kommander config file above.  

```bash
mkdir -p kaptain-catalog-applications 
tar -xvf ./kaptain-catalog-applications.tar.gz --strip-components 1 -C ./kaptain-catalog-applications
```

Then load the Kaptain charts after DKP installation with:

```bash
dkp push chart-bundle kaptain-charts-bundle.tar.gz
```